web/Gruntfile.js

```diff
@@ -40,8 +45,12 @@ module.exports = function (grunt) {
                     '<%= config.srcScriptsDir %>/**/*.es.js'
                 ],
                 tasks: [
-                    'compilejs'
+                    'build'
                 ]
+            },
+            ngtemplates: {
+                files: ['app/templates/**/*.html'],
+                tasks: ['build']
```

Comment:
Surely `dist` files should only be changed when you want to make a build?

If you wrote this for convenience when developing on it, the method of development should change, by either referencing the source in the project instead of dist, or you have a separate watcher that watches the source files used in a build and runs `build`.